File: euler12.py
def euler12():
    se=0
    temp=0
    for i in range(1,1000000,1):
        se+=i
        temp=0
        for j in range(1,int(se/2)+1):
            if se%j==0:
                temp+=1
        print("Tri: "+str(i)+" Divis: "+str(temp))
        if temp>=500:
            print(se)
            return i

print(euler12())

File: euler12.c
#include "CBP.h"

void euler12(){
    long long se=0;long long temp=0;
    for(int i=1;i<1000000;i++){
        se+=i;temp=0;
        for(int j=1;j<(int)(se/2)+1;j++){if(se%j==0){temp++;}}
        printf("\nTri: %d",i);printf(" Divis: %lld",temp);printf(" Value: %lld",se);
        if(temp>=500){printf(oint,i);printf(olld,se);break;}
    }
}

startMain
euler12();
endMain